tests/test_view_functions.py

```diff
@@ -60,6 +64,64 @@ def test_event_view_func_wrapper(monkeypatch):
     ]
 
 
+def test_cloudevent_view_func_wrapper(monkeypatch):
+    headers = {'Content-Type': 'application/cloudevents+json'}
+    data = json.dumps({
+        "source": "from-galaxy-far-far-away",
+        "type": "cloudevent.greet.you",
+        "specversion": "1.0",
+        "id": "f6a65fcd-eed2-429d-9f71-ec0663d83025",
+        "time": "2020-08-13T02:12:14.946587+00:00",
+        "data": {"name": "john"}
+    })
+
+    request = pretend.stub(headers=headers, get_data=lambda: data)
+    # event = from_http(request.get_data(), request.headers)
```

Comment:
Do you want to uncomment this and make assertions? Or do you want to delete this?

Reply:
I want to uncomment that if we manage to get out [cloudevents 1.0.1](github.com/cloudevents/sdk-python)

Reply:
https://github.com/cloudevents/sdk-python/pull/98